stations.csv

```diff
@@ -7535,7 +7535,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 9443;Annœullin;annoeullin;;;50.533333;2.933333;;FR;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;Аннёллен;;;阿讷兰;urn:trainline:public:nloc:csv9443;
 9444;Gondecourt;gondecourt;;;50.550000;2.983333;;FR;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;Гондекур;;;贡德库尔;urn:trainline:public:nloc:csv9444;
 9445;Monte-Carlo Country Club;monte-carlo-country-club;8733859;87338590;43.754354;7.448208;;FR;Europe/Paris;f;f;f;f;f;f;FRMCC;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv9445;
-9446;La Défense;la-defense;8720750;87207506;48.89147;2.24248;9817;FR;Europe/Paris;f;f;f;f;f;f;FRDGA;LDU;f;;f;8703563;f;;f;;f;;f;;f;;f;8703563;f;;f;;;;;f;;f;;f;;;f;;f;;f;;f;f;;Paris;Paris;Paris;Paris;Parigi;Paris;Parijs;Paříž;Paris;Párizs;パリ;파리;Paryż;Paris;Ля Дефанс;Paris;;巴黎;urn:trainline:public:nloc:csv9446;
+9446;La Défense;la-defense;8720750;87207506;48.89147;2.24248;9817;FR;Europe/Paris;f;f;f;t;f;f;FRDGA;LDU;f;;f;8703563;f;;f;@FRPARLDE;t;;f;;f;;f;8703563;f;;f;;;;;f;;f;;f;;;f;;f;;f;;f;f;;Paris;Paris;Paris;Paris;Parigi;Paris;Parijs;Paříž;Paris;Párizs;パリ;파리;Paryż;Paris;Ля Дефанс;Paris;;巴黎;urn:trainline:public:nloc:csv9446;
```

Comment:
🤔 We should probably call it `Paris La Défense`

Reply:
Anyways, it can be out of scope of this MR. Looking at the coordinates, this is indeed the correct station if we refer to the bus terminal.